src/2022.html

```diff
@@ -925,54 +925,6 @@ <h1 class="heading-1">TOPICS</h1>
         </div>
       </div>
       <div class="base-section">
-        <div id="section-creatives" class="base-div-full creatives">
-          <div class="base-container w-container">
-            <h1 class="heading-1">THE CREATIVES</h1>
-            <div class="subtitle">
-              LOCAL AND FAR, we gathered a few people that make our hearts and
-              feet beat
-            </div>
-          </div>
-          <div class="base-container w-container">
-            <div class="collection-list-wrapper w-dyn-list">
-              <div role="list" class="collection-list w-dyn-items">
-                <div role="listitem" class="collection-item w-dyn-item">
-                  <img
-                    src=""
-                    loading="lazy"
-                    alt=""
-                    class="base-image creatives w-dyn-bind-empty"
-                  />
-                  <div class="div-container">
-                    <h3 class="heading-3 creatives w-dyn-bind-empty"></h3>
-                    <img
-                      src="images/Speaker.svg"
-                      loading="lazy"
-                      alt="Speaker Icon"
-                      class="base-icon creatives"
-                    /><img
-                      src="images/Workshop.svg"
-                      loading="lazy"
-                      alt="Session Icon"
-                      class="base-icon creatives"
-                    /><img
-                      src="images/Performance.svg"
-                      loading="lazy"
-                      alt="Performance Icon"
-                      class="base-icon creatives"
-                    />
-                  </div>
-                  <div class="subtitle w-dyn-bind-empty"></div>
-                  <div class="divider short"></div>
-                  <div class="subtitle typed w-dyn-bind-empty"></div>
-                </div>
-              </div>
-              <div class="w-dyn-empty">
-                <div>No items found.</div>
-              </div>
-            </div>
-          </div>
-        </div>
```

Comment:
Not worth adding the 2022 speakers?

Reply:
Doesn't exist that content 🤔 
At least I didn't find on webflow, it was the same as 2023

> Remove section Creatives content from 2022 since, even on webflow, was duplicated with 2023

Reply:
Ahh I see - that sucks. Ok - right choice to remove it!